# ATAC-SEQ

### Datasets

1. https://www.10xgenomics.com/datasets/pbmc-from-a-healthy-donor-no-cell-sorting-10-k-1-standard-2-0-0
  PBMC from a Healthy Donor - Epi Multiome ATAC + Gene Expression dataset

2. https://www.10xgenomics.com/datasets/10-k-human-pbm-cs-multiome-v-1-0-chromium-x-1-standard-2-0-0

10k Human PBMCs, Epi Multiome ATAC + Gene Expression dataset          

 

Analysis questions

1) Run a quality control (QC), integrate/merge the datasets with each other, and cluster them. Based on RNA expression, what celltypes do you find? What clustering method would you use? Show why you made certain decisions/cutoffs for QC and labelling.
2) What are the functions of one or two cell types, based on a computational analysis?
3) Hypothetically, if dataset 2 would be a “diseased” patient, what downstream analysis/comparison would you suggest?
4) What else can you do with the single-cell RNA data?
5) Run or schematically show a CNV analysis (infercnv) and explain what it implies?
6) What would be your steps for an eQTL analysis and what is it? Simple workflow
7) If you have multiple pooled 10x multiome samples of different patients, how would you “unpool” them for downstream analysis? What data/information and tools would you need?
8) Utilizing pycisTopic, what does the ATAC data give you and can you combine it with the RNA data? A simple analysis/QC/clustering and a workflow for the combination is sufficient (if computational difficult, you can also just describe it).
9) For a spatial analysis e.g. 10x Xenium on our samples, what bioinformatic steps are crucial, to obtain good results?
 
